src/injector/src/main.rs

```diff
@@ -62,8 +62,14 @@ fn collect_binary_data(paths: &Vec<PathBuf>) -> io::Result<Vec<u8>> {
 ///
 /// Inspired by https://medium.com/@nlauchande/rust-coding-up-a-simple-concatenate-files-tool-and-first-impressions-a8cbe680e887
 fn unpack(sha_sum: &String) {
+    let init_root =
+        std::env::var("ZARF_INJECTOR_INIT_ROOT").unwrap_or_else(|_| String::from("/zarf-init"));
+    let seed_root =
+        std::env::var("ZARF_INJECTOR_SEED_ROOT").unwrap_or_else(|_| String::from("/zarf-seed"));
```

Comment:
These are necessary to make it possible to pass in a temporary directory location for testing purposes while still falling back to the default behavior when running as part of `zarf init`.